src/Collection/Collection.router.ts

```diff
@@ -146,4 +154,10 @@ export class CollectionRouter extends Router {
     ])
     return true
   }
+
+  async isCollectionNameAvailable(collection: CollectionAttributes) {
```

Comment:
I think receiving the entre collection as an argument is a bit overkill. What do you think about just getting the name here?

If we had another piece of code trying to check for a name it'd have to build an entire collection out of it

Reply:
Sounds reasonable